polypyus/cli.py

```diff
@@ -108,6 +109,46 @@ def _analyze(
         match_and_print(graph, target_path, parallelize)
 
 
+@orm.db_session
+@logger.catch
+def _cli_list(
+    list_history: bool, list_targets: bool,
+):
+    def format_binary_list(
+        binary_list: Iterable[Binary], is_annotated: Optional[bool] = False
+    ) -> DataFrame:
+
+        ignore_keys = ["is_target", "partitions", "raw"]
+        data = []
+        for binary in list(binaries):
+            b_dict = binary.to_dict()
+            if is_annotated is True:
+                b_dict["#annotations"] = len(binary.annotations)
+                b_dict["#functions"] = len(binary.annotations.functions)
+            data.append(b_dict)
+
+        df = DataFrame(data)
+        df = df.drop(ignore_keys, axis=1)
+        return df
```

Comment:
Question: I am not overly familiar with pandas dataframes. Does this drop method do anything that is worth adding a new dependency?

Reply:
If I understand correctly you want to exclude some attributes and you can do that already with 
to_dict method

https://docs.ponyorm.org/api_reference.html?highlight=to_dict#Entity.to_dict

Reply:
DataFrame is just a painless way to summarize multi-dimensional data. On it's own it already provides a nice print output, but I thought to use tabulate here for postprocessing to be consistent with the rest of the code.

 The `drop` function here is just to remove the keys which are not of interest - alternatively, one could pop() or del those items on the dict after generation and change the arguments to tabulate.

Reply:
The exclude kwarg to the to_dict() method helped here as well. I never worked with Pony ORM, so please excuse my lack of API knowledge ;)